security_drone_user_app/lib/presentation/pages/patrol_set_page.dart

```diff
@@ -32,16 +30,21 @@ class PatrolSetPageState extends State<PatrolSetPage> {
         padding: EdgeInsets.all(3),
         child: SizedBox(
           width: 75,
-          child: FlatButton(
-            padding: EdgeInsets.all(0),
+          child: TextButton(
+            //padding: EdgeInsets.all(0),
```

Comment:
Why did you change FlatButton to TextButton and removed the 0 padding?

Reply:
Saw a warning, it was suggested to implement the change.